sfe/sfe.go

```diff
@@ -105,9 +105,9 @@ func (sfe *SelfServiceFrontEndImpl) Handler(stats prometheus.Registerer, oTelHTT
 	m.Handle("GET /static/", staticAssetsHandler)
 	m.HandleFunc("/", sfe.Index)
 	m.HandleFunc("GET /build", sfe.BuildID)
-	m.HandleFunc(unpauseGetForm, sfe.UnpauseForm)
-	m.HandleFunc(unpausePostForm, sfe.UnpauseSubmit)
-	m.HandleFunc(unpauseStatus, sfe.UnpauseStatus)
+	m.HandleFunc(fmt.Sprint("GET "+unpauseGetForm), sfe.UnpauseForm)
+	m.HandleFunc(fmt.Sprint("POST "+unpausePostForm), sfe.UnpauseSubmit)
+	m.HandleFunc(fmt.Sprint("GET "+unpauseStatus), sfe.UnpauseStatus)
```

Comment:
This `fmt.Sprint` is a no-op. You're combining the two strings with a `+`, and then running that single string through `Sprint`.

```suggestion
	m.HandleFunc("GET "+unpauseGetForm, sfe.UnpauseForm)
	m.HandleFunc("POST "+unpausePostForm, sfe.UnpauseSubmit)
	m.HandleFunc("GET "+unpauseStatus, sfe.UnpauseStatus)
```